corehq/apps/cachehq/cachemodels.py

```diff
@@ -58,13 +58,12 @@ class UserRoleGenerationCache(GenerationCache):
 
 class ReportGenerationCache(GenerationCache):
     generation_key = '#gen#reports#'
-    doc_types = ['ReportConfig', 'HQGroupExportConfiguration', 'ReportNotification']
+    doc_types = ['ReportConfig', 'ReportNotification']
```

Comment:
wondering if this change could be related to http://manage.dimagi.com/default.asp?188411 somehow.  thoughts? @dannyroberts